locales/en/onboarding/onboarding_fingerprint.md

```diff
@@ -1 +1,2 @@
 **The biometric recognition via fingerprint is available in case you want to access the IO app faster**, without entering the unlock code every time. In order to use the biometric recognition, you need to enable it from the settings of the phone Operating System. You can always decide whether to keep the biometric recognition active or not in the IO app, by setting the preferences in the profile section… but we are pretty sure that once you try it, you won’t think of using the app without anymore.
+// edit
```

Comment:
Why do we have this comment? Because we are waiting for the copy review? I read "profile section" too a few line above.

Reply:
yes, are some comment for the copy review